components/visual-picker/__examples__/vertical-coverable.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import VisualPicker from '~/components/visual-picker';
+import { Radio, Checkbox } from '~/components';
+import Icon from '~/components/icon';
+import IconSettings from '~/components/icon-settings';
+
+class Example extends React.Component {
+	render() {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<div
+					style={{
+						padding: '4rem 1rem 0px',
+						background: 'rgb(244, 246, 249)',
+					}}
+				>
+					<h1>Radio</h1>
+					<VisualPicker
+						label="Select any one"
+						coverable
+						vertical
+						id="visual-picker-vertical-coverable-radio"
+					>
+						<Radio
+							id="visual-picker-vertical-coverable-radio-1"
+							onRenderVisualPickerSelected={
+								<Icon
+									assistiveText={this.props.assistiveText}
```

Comment:
This should have text in it. Currently it's undefined. The name of the icon would be fine since this example doesn't exist in SLDS.